source/command-monitoring/tests/unacknowledgedBulkWrite.yml

```diff
@@ -12,9 +12,10 @@ tests:
       name: "bulkWrite"
       arguments:
         requests:
-          - insertOne: 
+          - name: "insertOne"
+            arguments:
               document: { _id: "unorderedBulkWriteInsertW0", x: 44 }
-        ordered: false
+        options: { ordered: false }
```

Comment:
why use a different format in the YAML here? Shouldn't this just be:
```
options:
  ordered: false
```
?

Reply:
This is kosher YAML, and was already used elsewhere where the sub-object could easily be expressed on one line. It could also have been done for the `clientOptions` keys, since those are only defining a `retryWrites` option.